lib/dynamoid/associations/single_association.rb

```diff
@@ -19,16 +19,38 @@ def setter(object)
         object
       end
 
+      # Delete a model from the association.
+      #
+      #   post.logo.delete # => nil
+      #
+      # Saves both models immediately - a source model and a target one so any
+      # unsaved changes will be saved. Doesn't delete an associated model from
+      # DynamoDB.
       def delete
         target.send(target_association).disassociate(source.hash_key) if target && target_association
         disassociate
         target
       end
 
+
```

Comment:
Too many new lines.  Are we running Rubocop on this project?

Reply:
Rubocop rules are set up but we don't run it on CI. Only localy.